addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -703,7 +703,7 @@
                     <field name="company_id" optional="show" groups="base.group_multi_company" readonly="state != 'draft'"/>
                     <field name="activity_ids" widget="list_activity" optional="show" readonly="True"/>
                     <field name="journal_id" optional="hide" readonly="not is_editable"/>
-                    <field name="total_amount" sum="Total Amount" decoration-bf="1" widget="monetary"/>
+                    <field name="amount_residual" sum="Residual Amount" decoration-bf="1" widget="monetary"/>
```

Comment:
Please revert this one, that isn't what was intended.  We still want the total here (I agree that my original line in the task say that it's what we want and I don't know why)